engine/src/main/java/org/terasology/logic/console/commands/CoreCommands.java

```diff
@@ -517,13 +537,87 @@ public String spawnBlock(@Sender EntityRef sender, @CommandParam("blockName") St
+            "the player wants to drop"
+                , runOnServer = true, requiredPermission = PermissionManager.CHEAT_PERMISSION)
+    public String bulkDrop(@Sender EntityRef sender, @CommandParam("blockName") String blockName, @CommandParam("value") int value) {
+
+        //This is a loop which gives the particular amount of block the player wants to spawn
+        ClientComponent clientComponent = sender.getComponent(ClientComponent.class);
+        LocationComponent characterLocation = clientComponent.character.getComponent(LocationComponent.class);
+
+        Vector3f spawnPos = characterLocation.getWorldPosition();
+        Vector3f offset = characterLocation.getWorldDirection();
+
+        offset.scale(3);
+        spawnPos.add(5,10,0);
+        BlockFamily block = blockManager.getBlockFamily(blockName);
+        if (block == null)
+        {
+            return "Sorry, your block is not found";
+        }
+
+        BlockItemFactory blockItemFactory = new BlockItemFactory(entityManager);
+        if(value > 5000)
+        {
+            return "sorry the value you have entered exceeds the limit of 5000";
+        }
+
+        for (int i = 0; i <= value; i++) {
+
+                EntityRef blockItem = blockItemFactory.newInstance(block);
+                blockItem.send(new DropItemEvent(spawnPos));
+        }
+
+        // this returns the block you have spawned and the amount
+        return "Dropped " + value + " " + blockName + " Blocks :)";
+    }
+
+    @Command(shortDescription = "Sets up a typical bowling pin arrangement in front of the player. ", helpText = "Spawns a specific blocks in a regular bowling pin pattern, The item " +
+            " front of the player can simply picked up",
+            runOnServer = true, requiredPermission = PermissionManager.CHEAT_PERMISSION)
+    public String bowlingPrep  (@Sender EntityRef sender, @CommandParam("blockName") String blockName) {
+
+        ClientComponent clientComponent = sender.getComponent(ClientComponent.class);
+        LocationComponent characterLocation = clientComponent.character.getComponent(LocationComponent.class);
+
+        Vector3f spawnPos = characterLocation.getWorldPosition();
+        Vector3f offset = characterLocation.getWorldDirection();
+        offset.scale(5);
+        spawnPos.add(offset);
+        BlockFamily block = blockManager.getBlockFamily(blockName);
+        if (block == null) {
+            return "Sorry, your block is not found";
+        }
+
+        BlockItemFactory blockItemFactory = new BlockItemFactory(entityManager);
+        Vector3f startPos = new Vector3f(spawnPos);
+
+        float deltax = .2f; // delta x is the distance between the pins in vectorX coordinate
+        float deltaz = .3f; //delta z is the distance between the pins in vectorZ coordinate
```

Comment:
For clarity it may help to indicate what dimension `z` is rather than just saying delta z is the dist between pins in vectorZ - that somewhat just rephrases things :)